### DISCUSSION TITLE
SAST vs DAST scanning for code security

### DISCUSSION THREAD

**Discussion by steve-flexagon** [FlexDeploy-Community/community#28] [Show and tell]
Did you know about the difference between SAST and DAST scanning for code security?

SAST (Static application security testing) analyzes code for security vulnerabilities before the code have been compiled. Since the scan has run at build time, security vulnerabilities can be found before they make it down you CI/CD pipeline.

DAST (Dynamic Application Security Testing) is the process of analyzing an application at run time. More severe issues can be found as most scans use simulated attacks to scan for vulnerabilities.

Learn more about scanning in FlexDeploy [Here ](https://www.youtube.com/watch?v=E6R-Pal-y_w)

**Comment by jungaretti** [comment]
Wow, that’s sick!!
